python/README.md

```diff
@@ -1251,7 +1251,33 @@ deactivate
 ```
 If you want to re-enter the virtualenv just follow the same instructions above about activating a virtualenv. There’s no need to re-create the virtualenv.
 
-##References
+### VirtualenvWrapper
+Even tough is not mandatory, there is a wrapper that sets some defaults about virtualenv use.
+You can install it with pip following this instructions.
```

Comment:
Suggestion, maybe would be good to clarify: virtualenvwrapper should be installed into the same global site-packages area where virtualenv is installed. You may need administrative privileges to do that.
You should only ever install virtualenvwrapper on your base Python installation (i.e. NOT while a virtualenv is active)